e2e-polybft/framework/test-cluster.go

```diff
@@ -51,6 +51,15 @@ const (
 	nonValidatorPrefix = "test-non-validator-"
 )
 
+const (
+	Validator = 1 << iota
+	Relayer
+)
```

Comment:
Can we include `None` as a value as well? In that case, we would use that one to describe if the node is neither a validator nor a relayer

The full example is here: https://go.dev/play/p/DhzDEClIROV

```suggestion
type NodeType int

const (
	None      = 0
	Validator = 1 << iota
	Relayer
)

func (nt NodeType) IsSet(value NodeType) bool {
	return nt&value == value
}
```